### DISCUSSION TITLE
Live preview on Section Entry Types

### DISCUSSION THREAD

**Discussion by seamusmh** [craftcms/cms#12599] [Ideas]
I like the UI for organizing Entry Types better than the drag & drop interfaces I've used in other systems. However, I often forget exactly what's in each field and/or what field types I'm adding (here I am trying to figure out which hours field to use). And it's nice to see how 1/4 & 1/2 width blocks fit on the page. Would love to have live preview functionality here:

<img width="1521" alt="Screen Shot 2023-01-31 at 11 10 53 AM" src="https://user-images.githubusercontent.com/76339/215860322-b6862c52-176a-40c5-85a7-3333dac804d7.png">

**Comment by thupsi** [comment]
That's an interesting idea! 

I often go back and forth, rearranging the fields, reloading an entry in another tab to check, etc. Having something like live preview would be a major convenience for sure!
